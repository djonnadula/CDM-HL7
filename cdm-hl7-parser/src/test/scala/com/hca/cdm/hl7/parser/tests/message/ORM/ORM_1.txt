MSH|^~\&|MT_FRNK|FRNK|DBM||201702090828||ORM^O01|MT6_FRNK_ORM_156056082|P|2.5.1||KYB|||
PID|1|A00009619|K000100072^^^^MR^FRNK~400-96-5317^^^^SS^FRNK~K147602^^^^PI^FRNK~A00008619^^^^HUB^FRNK|K147672|SMALLEEBURK^GWANDOLYN^MCCALL^^^^L~SMALLENBURG^GWENOOLYN^J^^^^M|CUROL|19600210|F||W|122 IMKEUIQQ AVE^^FRANKFORT^KY^49601||(502)220-6988|||M|BAP|FR0804565133
PV1|1|E|FR.ER|EM|||PZI7367^Walker^Forest^^^^MD|SELF^Referred^Self||||||PR||||ER||13|||||||||||||||||||FRNK||REG|||201702090633
AL1|1|DA|F001900388^No Known Allergies^^No Known Allergies^^allergy.id|||20170209
ORC|NW|FR000370672~17-8049FRNK|||L|N|||201702090828|PZI7367|||FR.ER
OBR|1|FR000370672~17-8049FRNK||RAD^CHE2^Chest 2 View||201702090828|||||||FR|||PZI7367^Walker^Forest^^^^MD|(502)875-5240^^PH^^^502^875-5240|RAD^20170209-0025||DI|||||||1^^^201702090828^^S|PZI7367^Walker^Forest^^^^MD~JBC4496^Haddix^Allen^C^^^MD~SELF^Referred^Self|||1ORD01004A^cough, soa|PZI7367
OBX|1|TX|1ORD01004A^Reason for Exam||^cough, soa
OBX|2|TX|1ORD01005A^Comment
OBX|3|TX|1ORD01050A^Portable?||^N