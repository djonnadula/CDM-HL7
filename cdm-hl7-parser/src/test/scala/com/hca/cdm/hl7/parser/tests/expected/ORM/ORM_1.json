{
  "0000.COMN": {
    "sending_facility": "FRNK",
    "message_control_id": "MT6_FRNK_ORM_156056082",
    "medical_record_num": "K012310231",
    "medical_record_urn": "K147672",
    "patient_account_num": "FR0804565133"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "MT_FRNK",
    "sending_facility": "FRNK",
    "receiving_application": "DBM",
    "receiving_facility": null,
    "date_time_of_message": "201412120828",
    "security": null,
    "message_type": {
      "message_code": "ORM",
      "trigger_event": "O01"
    },
    "message_control_id": "MT6_FRNK_ORM_156056082",
    "processing_id": "P",
    "version_id": "2.5.1",
    "sequence_num": null,
    "continuation_pointer": "KYB"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id": "A00009619",
    "patient_id_list": [
      {
        "id_num": "K012310231",
        "id_check_digit": null,
        "check_digit_scheme": null,
        "assigning_authority": null,
        "id_type_code": "MR",
        "assigning_facility": "FRNK"
      },
      {
        "id_num": "999-99-9999",
        "id_check_digit": null,
        "check_digit_scheme": null,
        "assigning_authority": null,
        "id_type_code": "SS",
        "assigning_facility": "FRNK"
      },
      {
        "id_num": "K1513251",
        "id_check_digit": null,
        "check_digit_scheme": null,
        "assigning_authority": null,
        "id_type_code": "PI",
        "assigning_facility": "FRNK"
      },
      {
        "id_num": "A12412312",
        "id_check_digit": null,
        "check_digit_scheme": null,
        "assigning_authority": null,
        "id_type_code": "HUB",
        "assigning_facility": "FRNK"
      }
    ],
    "alternate_patient_id": "K147672",
    "patient_name": [
      {
        "family_name": "SMALLEEBURK",
        "given_name": "GWANDOLYN",
        "second_and_further_given_names_or_initials_thereof": "MCCALL",
        "suffix": null,
        "prefix": null,
        "degree": null,
        "name_type_code": "L"
      },
      {
        "family_name": "SMALLENBURG",
        "given_name": "GWENOOLYN",
        "second_and_further_given_names_or_initials_thereof": "J",
        "suffix": null,
        "prefix": null,
        "degree": null,
        "name_type_code": "M"
      }
    ],
    "mother_maiden_name": "CUROL",
    "date_time_of_birth": "19600210",
    "administrative_sex": "F",
    "patient_alias": null,
    "race": "W",
    "patient_address": {
      "street_address": "122 IMKEUIQQ AVE",
      "other_designation": null,
      "city": "FRANKFORT",
      "state_or_province": "KY",
      "zip_or_postal_code": "49601"
    },
    "county_code": null,
    "phone_num_home": "(999)999-9999",
    "phone_num_business": null,
    "primary_language": null,
    "marital_status": "M",
    "religion": "BAP",
    "patient_account_num": "FR0804565133"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "E",
    "assigned_patient_location": "FR.ER",
    "admission_type": "EM",
    "pre_admit_num": null,
    "prior_patient_location": null,
    "attending_doctor": {
      "person_id": "PZI7367",
      "family_name": "Walker",
      "given_name": "Forest",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": "MD"
    },
    "referring_doctor": {
      "person_id": "SELF",
      "family_name": "Referred",
      "given_name": "Self"
    },
    "consulting_doctor": null,
    "hospital_service": null,
    "temporary_location": null,
    "pre_admit_test_indicator": null,
    "readmission_indicator": null,
    "admit_source": "PR",
    "ambulatory_status": null,
    "vip_indicator": null,
    "admitting_doctor": null,
    "patient_type": "ER",
    "visit_num": null,
    "financial_class": "13",
    "charge_price_indicator": null,
    "courtesy_code": null,
    "credit_rating": null,
    "contract_code": null,
    "contract_effective_date": null,
    "contract_amt": null,
    "contract_period": null,
    "interest_code": null,
    "transfer_to_bad_debt_code": null,
    "transfer_to_bad_debt_date": null,
    "bad_debt_agency_code": null,
    "bad_debt_transfer_amt": null,
    "bad_debt_recovery_amt": null,
    "delete_account_indicator": null,
    "delete_account_date": null,
    "discharge_disposition": null,
    "discharged_to_location": null,
    "diet_type": null,
    "servicing_facility": "FRNK",
    "bed_status": null,
    "account_status": "REG",
    "pending_location": null,
    "prior_temporary_location": null,
    "admit_date_time": "201412120633"
  },
  "0004.AL1": {
    "set_id": "1",
    "allergen_type_code": "DA",
    "allergen_code_mnemonic_description": {
      "id": "F001900388",
      "text": "No Known Allergies",
      "name_of_coding_system": null,
      "alternate_id": "No Known Allergies",
      "alternate_text": null,
      "name_of_alternate_coding_system": "allergy.id"
    },
    "allergy_severity_code": null,
    "allergy_reaction_code": null,
    "identification_date": "20141212"
  },
  "0005.ORC": {
    "order_control": "NW",
    "placer_order_num": "FR000370672~17-8049FRNK",
    "filler_order_num": null,
    "placer_group_num": null,
    "order_status": "L",
    "response_flag": "N",
    "quantity_timing": null,
    "parent_order": null,
    "date_time_of_transaction": "201412120828",
    "entered_by": "PZI7367",
    "verified_by": null,
    "ordering_provider": null,
    "enterer_location": "FR.ER"
  },
  "0006.OBR": {
    "set_id": "1",
    "placer_order_num": "FR000370672~17-8049FRNK",
    "filler_order_num": null,
    "universal_service_id": {
      "id": "RAD",
      "text": "CHE2",
      "name_of_coding_system": "Chest 2 View"
    },
    "priority": null,
    "requested_date_time": "201412120828",
    "obsv_date_time_num": null,
    "obsv_end_date_time_num": null,
    "collection_volume": null,
    "collector_id": null,
    "specimen_action_code": null,
    "danger_code": null,
    "relevant_clinical_information": "FR",
    "specimen_received_date_time": null,
    "specimen_source": null,
    "ordering_provider": {
      "person_id": "PZI7367",
      "family_name": "Walker",
      "given_name": "Forest",
      "second_and_further_given_names_or_initials_thereof": null,
      "suffix": null,
      "prefix": null,
      "degree": "MD"
    },
    "order_callback_phone_num": {
      "telephone_num": "(999)999-9999",
      "telecommunication_use_code": null,
      "telecommunication_equipment_type": "PH",
      "communication_address": null,
      "country_code": null,
      "area_city_code": "502",
      "local_num": "875-5240"
    },
    "placer_field_1": {
      "order_category": "RAD",
      "order_number": "20141212-0025"
    },
    "placer_field_2": null,
    "filler_field_1": "DI",
    "lab_site": null,
    "results_rpt_status_chng_date_time": null,
    "charge_to_practice": null,
    "diagnostic_service_section_id": null,
    "result_status": null,
    "parent_result": null,
    "quantity_timing": {
      "quantity": "1",
      "interval": null,
      "duration": null,
      "start_date_time": "201412120828",
      "end_date_time": null,
      "priority": "S"
    },
    "result_copies_to": [
      {
        "person_id": "PZI7367",
        "family_name": "Walker",
        "given_name": "Forest",
        "second_and_further_given_names_or_initials_thereof": null,
        "suffix": null,
        "prefix": null,
        "degree": "MD"
      },
      {
        "person_id": "JBC4496",
        "family_name": "Haddix",
        "given_name": "Allen",
        "second_and_further_given_names_or_initials_thereof": "C",
        "suffix": null,
        "prefix": null,
        "degree": "MD"
      },
      {
        "person_id": "SELF",
        "family_name": "Referred",
        "given_name": "Self"
      }
    ],
    "parent_results_obsv_id": null,
    "transportation_mode": null,
    "reason_for_study": {
      "id": "1ORD01004A",
      "text": "cough, soa"
    },
    "principal_result_interpreter": "PZI7367"
  },
  "0007.OBX": {
    "set_id": "1",
    "value_type": "TX",
    "obsv_id": {
      "id": "1ORD01004A",
      "text": "Reason for Exam"
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": "cough, soa"
    }
  },
  "0008.OBX": {
    "set_id": "2",
    "value_type": "TX",
    "obsv_id": {
      "id": "1ORD01005A",
      "text": "Comment"
    }
  },
  "0009.OBX": {
    "set_id": "3",
    "value_type": "TX",
    "obsv_id": {
      "id": "1ORD01050A",
      "text": "Portable?"
    },
    "obsv_sub_id": null,
    "obsv_value": {
      "id": null,
      "text": "N"
    }
  }
}